CHANGELOG.next.toml

```diff
@@ -485,3 +485,108 @@ message = "Make generated enum `values()` functions callable in const contexts."
 references = ["smithy-rs#2011"]
 meta = { "breaking" = false, "tada" = false, "bug" = false, "target" = "all" }
 author = "lsr0"
+
+[[smithy-rs]]
+message = """
+# Plugins/New Service Builder API
```

Comment:
Will an h1 title be rendered ok in the changelog?

I ask because changelog items are displayed in bullet points see e.g. https://github.com/awslabs/smithy-rs/releases/tag/release-2022-08-31

Reply:
There's two others ("New Documentation", "Deprecations") below.

Reply:
Good question. It looks like it works here:

# H1 A

- Some text
  # H1 B
  Some more text
  # H1 C
  Some more text
  ## H2
  Some more text
- Some more text
   # H1 D

and I'd be suprised if it was rendered differently in the release page.

I can change the headers to **bold** if you'd like?

Reply:
It works in the release page. You can try it out [here](https://github.com/awslabs/smithy-rs/releases/edit/release-2022-08-31). But the h1 is bigger than the "Breaking Changes" heading. An h3 `###` looks nicer.

![image](https://user-images.githubusercontent.com/6341745/205342257-bc6fe5c3-0da4-4af6-b615-61c5b674f178.png)

Reply:
Will change to H3 👍